Action/ConvertPaymentAction.php

```diff
@@ -41,6 +41,7 @@ public function execute($request)
             'email' => $order->getClientEmail(),
             'firstName' => isset($d['firstName']) ? $d['firstName'] : '',
             'lastName' => isset($d['lastName']) ? $d['lastName'] : '',
+            'language' => $order->getLocale() ? substr($order->getLocale(), 0, 2) : '',
```

Comment:
I think we should not force people to add this method - it's not present in any interface and adding it would cause BC break. 

We can introduce a separate interface and check if it is implemented. We could also store constants with available language codes in it.
I will check how other gateways approached this issue.